bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -51,16 +53,28 @@ const setLoading = (val) => {
   }
 };
 
-export default createContainer(() => {
+const checkUnreadMessages = () => {
+  return userListService.getOpenChats().map(chat=> chat.unreadCounter)
+                        .filter(userID => userID !== Auth.userID);
+};
+
+export default createContainer(({ params }) => {
   Promise.all(subscribeForData())
   .then(() => {
     setLoading(false);
   });
 
+  const openChats = () => {
```

Comment:
this was initially written as return {.... openChats: userListService.getOpenChats(params.chatID).map(chat => chat.id), .......}
Would keeping it that way be an option?